common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -208,31 +198,11 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
-    "ab-acquisitions-epic-aus-fairfax",
-    "Tests an epic with custom copy in Australia which alludes to Fairfax Media takeover",
-    owners = Seq(Owner.withGithub("joelochlann")),
-    safeState = Off,
-    sellByDate = new LocalDate(2019, 6, 6),
-    exposeClientSide = true
-  )
-
-  Switch(
-    ABTests,
-    "ab-acquisitions-epic-one-million-campaign-no-control",
-    "Tests an epic with custom copy and design for the One Million campaign",
-    owners = Seq(Owner.withGithub("joelochlann")),
-    safeState = Off,
-    sellByDate = new LocalDate(2018, 11, 23),
-    exposeClientSide = true
-  )
-
-  Switch(
-    ABTests,
-    "ab-acquisitions-epic-one-million-campaign-au",
-    "Tests an epic with custom copy and design for the One Million campaign",
+    "ab-acquisitions-epic-au",
```

Comment:
I think this test name should be more specific. The testing framework (and analysis, which is probably a bigger concern as you may name a test the same thing as a test 2 months ago, which a data analyst wouldn't be aware of) depends on the test names being distinct.